packages/code-sandbox/src/SandboxApp.tsx

```diff
@@ -26,7 +26,7 @@ const SandboxApp: React.FunctionComponent = props => {
   const [theme, setTheme] = React.useState(items[0].value)
 
   return (
-    <Provider theme={themes[theme]} styles={{ height: '100vh', padding: '1rem' }}>
+    <Provider theme={themes[theme]} styles={{ padding: '0 0 0 1rem' }}>
```

Comment:
Have you tried this with other themes?

Reply:
![image](https://user-images.githubusercontent.com/10930160/63699658-90c7cd00-c821-11e9-88db-feade3407820.png)

Reply:
It works now
![image](https://user-images.githubusercontent.com/10930160/63700191-9c67c380-c822-11e9-824f-3b2762929073.png)